### ISSUE TITLE
Comments: when post is missing a title use excerpt or alt text

### ISSUE THREAD

**Issue by gwwar** [Automattic/wp-calypso#15455]
This is a follow up to https://github.com/Automattic/wp-calypso/pull/15315#discussion_r123341546

Sometimes posts do not have titles. Instead of displaying `untitled`, let's try to get a short except of the post content. If the post only contains an image, try to use it's alt text. If a post has no text or alt text, fallback to `untitled`
